### PR TITLE
0.2/665 fuzzy search medium

### PR DESCRIPTION
**Pull request by charlesstrange2**

### CODE REVIEW

File: resources/views/service/index_registration.blade.php [hunk 1]
```diff
@@ -40,7 +40,7 @@
                     @foreach ($registrations as $registration)
                         <tr>
                             <td>{{ $registration->family->carers->first()->name }}</td>
-                            <td>{{ $registration->cc_reference }}</td>
+                            <td>{{ $registration->family->rvid }}</td>
```
Comment (alexalala) [inline resources/views/service/index_registration.blade.php]: From prototyping, voucher entitlement is the second col in the div and then RV ID. As I've already changed this, could just remove the changes to this file?

**Review by alexalala — CHANGES_REQUESTED** [review]
Limited Laravel knowledge, but the bits I do understand I dont have a problem with! Other than the page I've commented on 👍  will approve when it's amended.

**Comment by charlesstrange2** [comment]
Agreed, will amend.

**Review by alexalala — APPROVED** [review]
Beautiful! 🎊

**Pull request merged**